main.py

```diff
@@ -108,15 +102,6 @@ def main():
     with WATCH_HISTORY_FILE.open(encoding="utf8") as f:
         data = json.load(f)
 
-    if REMOVE_SHORTS is True:
```

Comment:
@Indranil012 Can you open a separate PR to filter shorts? These are not necessary.

Reply:
Ok